packages/ng/src/app/select/select.component.html

```diff
@@ -2,12 +2,10 @@
 	[class.mod-remove]="_canRemove"
 	[luSelect]="selectRef">
 	<div
-		class="textfield-input"
-		>{{_strValue}}</div>
+	class="textfield-input"
+	>{{_strValue}}</div>
+	<ng-content></ng-content>
 	<lu-select-picker #selectRef (itemSelected)="_optionSelected($event)">
-		<ng-content></ng-content>
+		<ng-content select="[option],lu-select-option"></ng-content>
```

Comment:
wot?

Reply:
you can do like materail and get all the options via a `viewcontentchildren`

Reply:
I'm really open to new way to write this. The idea behind this selector is to specify a separation between content children of type clear / search and the others options that have to come in the content of picker.

The problem was that, i don't find a way to specify `select="EveryThingsThatInheritFromLuSelectOption"` the selector is a css selector so there is no consideration to the type of options :'(. So i want to specify that if you want to use something different of `lu-select-option` you have to specify something that have the `[option]`attribute.

But, I'm asking me, if I should not refactor the actual lu-select-option and set the value of option with `luOptionValue` instead of `value` and change the selector in my select component to `[luOptionValue]`

What did you about this ?

**Actually**
```
<lu-select>
   <lu-select-option [value]="option" *ngFor="option in options">{{option.name}}</lu-select-option>  
</lu-select>

<!-- And the selector in select.component.html -->
<ng-content select="[option],lu-select-option"></ng-content>
```

**Proposition**
```
<lu-select>
   <lu-select-option [luOptionValue]="option" *ngFor="option in options">{{option.name}}</lu-select-option>  
</lu-select>

<!-- And the selector in select.component.html -->
<ng-content select="[luOptionValue]"></ng-content>
```

Reply:
both synthaxes work for me as the end user will always just write
```html
<lu-select>
   <lu-select-option [thePropYouWant]="option" *ngFor="option in options">{{option.name}}</lu-select-option>  
</lu-select>
```